src/workerd/api/web-socket.h

```diff
@@ -425,6 +425,9 @@ class WebSocket: public EventTarget {
   // `close()`, thereby preventing calls to `send()` even after we wake from hibernation.
   bool closedOutgoingForHib = false;
 
+  // Maximum allowed size for WebSocket messages
+  size_t maxMessageSize = 1u << 20;
```

Comment:
Instead of having this be a member of `webSocket`, could you pass it as a parameter to `readLoop()` (reading the feature flag in `startReadLoop()`)?

Reply:
https://github.com/cloudflare/workerd/pull/2164/commits/e1fcf20a31aa5617c3e2a253634e84dd7c82fa08